src/contrib/loggraph/src/java/org/apache/zookeeper/graph/TxnLogSource.java

```diff
@@ -76,12 +76,14 @@ public boolean overlapsRange(long starttime, long endtime) {
     public LogSkipList getSkipList() { return skiplist; }
 
     public static boolean isTransactionFile(String file) throws IOException {
-        RandomAccessFileReader reader = new RandomAccessFileReader(new File(file));
-        BinaryInputArchive logStream = new BinaryInputArchive(reader);
-        FileHeader fhdr = new FileHeader();
-        fhdr.deserialize(logStream, "fileheader");
-	reader.close();
-
+    	try {
+    		RandomAccessFileReader reader = new RandomAccessFileReader(new File(file));
```

Comment:
spaces, but historically this hasn't been consistently enforced. it's getting better tho :-)